nixos/modules/services/networking/firewall.nix

```diff
@@ -506,14 +507,12 @@ in
     environment.systemPackages = [ pkgs.iptables ] ++ cfg.extraPackages;
 
     boot.kernelModules = map (x: "nf_conntrack_${x}") cfg.connectionTrackingModules;
-    boot.extraModprobeConfig = optionalString (!cfg.autoLoadConntrackHelpers) ''
-      options nf_conntrack nf_conntrack_helper=0
+    boot.extraModprobeConfig = optionalString cfg.autoLoadConntrackHelpers ''
+      options nf_conntrack nf_conntrack_helper=1
```

Comment:
@fpletz  Did you test that change with a reboot? I'm not entirely sure about this, but I also wasn't entirely sure why it worked before :D

My guess: Before we loaded `nf_conntrack_ftp` which required `nf_conntrack` and resulted in `nf_conntrack` being loaded. This would also explain why we're loading `nf_conntrack` if `cfg.autoLoadConntrackHelpers == false`.

But since we now aren't loading `nf_conntrack_ftp` anymore and only load `nf_conntrack` if `cfg.autoLoadConntrackHelpers` is set it might be possible that we won't be loading `nf_conntrack` which would break the firewall (as far as I know).

However it might still work if we load that module elsewhere or if we load another module that depends on it (e.g `xt_conntrack`, `nf_conntrack_ipv4` (both are currently loaded on my system).

But if I didn't miss anything it might be a good idea to load `nf_conntrack` in any case.